task/sumOfOther.js

```diff
@@ -0,0 +1,8 @@
+function sumOfOther(arr) {
+  const result = arr.map((elem, i, array) => array.reduce((accum, current, j) => (
+    i !== j ? accum + current : accum
```

Comment:
You have showed deep understanding of array's methods working, but... you have lost something :) now you have O(n2) complexity instead 2O(n).